cellpack/autopack/upy/hostHelper.py

```diff
@@ -5028,6 +5030,26 @@ def writeToFile(self, polygon, filename):
             filename, verts=vertices, faces=faces, vnorms=vnormals, fnorms=fnormals
         )
 
+    @classmethod
+    def saveDejaVuMesh(self, filename, verts=[], faces=[]):
```

Comment:
I think that the convention for classmethods is that the first argument is `cls` instead of `self1